checkout/automatic.php

```diff
@@ -0,0 +1,25 @@
+<!--  The entire list of Checkout fields is available at
+ https://docs.razorpay.com/docs/checkout-form#checkout-fields -->
+
+<form action="verify.php" method="POST">
+  <script
+    src="https://checkout.razorpay.com/v1/checkout.js"
+    data-key="<?php echo $keyId?>"
+    data-amount="<?php echo $amount?>"
+    data-currency="INR"
+    data-name="Daft Punk"
+    data-description="Purchase Description"
+    data-image="../daft-punk.jpg"
```

Comment:
still using local image link. We need to pick as many values as we can from the data that was passed.